src/insights.jsx

```diff
@@ -587,7 +540,7 @@ export class InsightsStatus extends React.Component {
                             target="_blank" rel="noopener noreferrer"
                             icon={<ExternalLinkAltIcon />}
                         >
-                            { text }
+                            { _("View your Insights results") }
```

Comment:
IMHO we shouldn't put that on the Health card -- that is meant for concrete, actionable information. This is just a constant now, so the user does not know whether anything needs attention. Requesting and providing that information (affected rule hits for  a agiven machine) is its core functionality. Is that available through a different API?

If not, let's just entirely remove the InsightsStatus class and status integration.

Reply:
No, there is no different API that could provide such information.

What do you mean with this?

> If not, let's just entirely remove the InsightsStatus class and status integration.

Do you mean that  you want to remove "insights" from Health card? if yes, then it is anything we can solve in this PR, right?

Reply:
Yes, cockpit implements the health card insights integration itself, so we'll do that on the cockpit side. What I meant is, showing that static string on the Subscriptions page isn't very useful either IMHO. There must be *some* way to ask Insights "how many/which things affect my system?" surely? I mean, that's what it does?